inc/class-markdownparser.php

```diff
@@ -165,6 +170,12 @@ protected function blockHeader( $data ) {
 		$block = parent::blockHeader( $data );
 		$id = sanitize_title_with_dashes( $block['element']['text'] );
 
+		// Use a manual ID if provided.
+		if ( preg_match( '/^(.+) ?\{#(.+?)\}$/', $block['element']['text'], $matches ) ) {
```

Comment:
Will be good to update the actual module docs with this feature too

Reply:
Ooh yeah